.github/workflows/ci.yml

```
@@ -19,9 +19,9 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: ${{ env.NODE_VERSION }}
-          cache: 'yarn'
-      - run: yarn install --frozen-lockfile
-      - run: yarn lint
+          cache: 'npm'
+      - run: npm install --frozen-lockfile
```

```suggestion
      - run: npm ci
```

If I'm not mistaken this command does not work this way with `npm`